tests/integration/linux_bridge_test.py

```diff
@@ -396,9 +396,9 @@ def test_port_vlan_not_implemented(port0_up):
     }
 
     bridge_state = _create_bridge_subtree_config((port_name,))
-    bridge_state[LinuxBridge.PORT_SUBTREE][0].update(port_vlan_config)
-    with linux_bridge(bridge_name, bridge_state):
```

Comment:
It seems to be strange that this is needed. It seems that either `linux_bridge()` should be fixed or there is a bug in Nmstate.

If libnmstate.apply() fails, then Nmstate should revert everything with a rollback. Otherwise `linux_bridge()` should do it.

Reply:
I am trying to reproduce it out of the test. I don't see anything wrong on linux_bridge or nmstate code..

Reply:
What are you seeing inside of tests?